pandas/tests/indexing/test_scalar.py

```diff
@@ -9,61 +9,59 @@
 
 
 class TestScalar(Base):
-    def test_at_and_iat_get(self):
+    @pytest.mark.parametrize("kind", ["series", "frame"])
```

Comment:
make sure we use the same nomenclature, usually we call this klass and use the object itself